[WIP] Outline the Study section

agitter opened the pull request:
I outlined initial thoughts about what topics to cover and which we might de-emphasize or leave out.  Before merging, let's discuss and finalize the sub-sections so that we can divide them and begin writing.

Tagging contributors from #116 who expressed interest in this section: @traversc @w9 @kumardeep27 @gokceneraslan @gwaygenomics @minseven @gailrosen @XieConnect

gailrosen commented:
I think I'll have enough for about 3-4 paragraphs under metagenomics... is that enough?

agitter commented:
@gailrosen Yes, I'd say so.  Can you please add a few bullet points in the comments here about the main messages?  Then we can see how it fits with the rest of the section, and I can merge it into this pull request.

gailrosen commented:
I guess the main points will revolve around some of the pros and cons I have found.  But this is what I can find that's remotely related to metagenomics.

* Large-scale machine Learning has been applied to metagenomics problems to understand genotype-phenotype relationships and taxonomy:
Segata: http://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1004977 
Guetterman:  http://www.fasebj.org/content/30/1_Supplement/406.3.short 
Vervier: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896366/ 

* Multi-layer neural networks have been applied to homology and protein classification problems (will have to go over pros and cons):
Hochreiter:  https://www.ncbi.nlm.nih.gov/pubmed/17488755 
Asgari E, Mofrad MR. Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics. PloS one 2015;10(11):e0141287. 
Sønderby SK, Sønderby CK, Nielsen H et al. Convolutional LSTM Networks for Subcellular Localization of Proteins. arXiv preprint arXiv:1503.01919 2015.

* Neural Networks are helping to denoise new sequencing technologies (nanopore, pros and cons)  https://nanoporetech.com/publications/deepnano-deep-recurrent-neural-networks-base-calling-minion-nanopore-reads 

* Only recently have multi-layer and/or recurrent neural and/or convolutional networks applied to microbiome genotype-phenotype (pros and cons):
http://ieeexplore.ieee.org/document/7219432/
http://www.sciencedirect.com/science/article/pii/S1413867015001889
http://alifar76.github.io/sklearn-metrics/ 

* And are now being explored for bacterial sequence classification:
http://eprints.fri.uni-lj.si/3573/1/63120248-NINA_MRZELJ-Globoko_u%C4%8Denje_na_genomskih_in_filogenetskih_podatkih.pdf 

Microbiome machine learning challenge: https://github.com/alifar76/TFMicrobiome
And potential

XieConnect commented:
@agitter  The first half of subsections seems good. Since I'm less up-to-date with the second half, I'll wait for comments from other experts. 

Regarding my responsibility now: do you suggest I also list bullet points about main messages/works beneath the subsections I'm familiar with? Some these topics have been included in other reviews, so I need to figure out ways to differentiate as needed.

sw1 commented:
Three overlapping papers: 2 on deep exponential families and 2 on deep survival analysis:

1. Deep Exponential Families (http://arxiv.org/abs/1411.2581): A non-linear deep architecture that builds on exponential family distributions and hence an be applied to a variety of data types (e.g., binary data, count data). They provide examples that are extensions of sigmoid belief networks and make additoinal comparisons to RBMs. The model is fit with variational information, so that might be another opportunity to tie it with other models.

2. Deep Survival Analysis (http://arxiv.org/abs/1608.02158): An application of DEFs where they predicted disease risk using EHRs. See https://github.com/greenelab/deep-review/issues/81

3. Deep Survival: A Deep Cox Proportional Hazards Network (http://arxiv.org/abs/1606.00931): Another deep survival model, but instead of using DEFs, this is a deep implementation of the Faraggi-Simon survival network. The model predicts disease risk, as well as acting as a recommender system in that it can provide the risk of being on treatment A compared to B. A variety of older (shallow) architectures are described, so making comparisons would be easy. They used real and simulated data.

agitter commented:
@gailrosen That looks like plenty of material.  I haven't read those papers so I still don't have a good sense of what those pros and cons are or how subcellular localization fits with metagenomics.  Feel free to propose a reorganization of the sub-sections if there is a better way to approach this.

As the next step, can you please suggest topic sentences or bullet points about the pros and cons for these paragraphs?  Then I can add them to my pull request, and you'll be able to start writing this section.

agitter commented:
@XieConnect Yes, I would like to move forward with main messages for these topics.  I can contribute but would like help from others.  For those topics that have already been covered well in other reviews, I suggest we plan to keep our text short for now.  As an example, predicting the effects of non-coding variants on TF binding has been discussed in #47 and elsewhere.  We could refer to the review and primary sources, very briefly recap the models, and focus our discussion on any particular new commentary, critiques, or future opportunities.  We can also emphasize the connection to human disease.

agitter commented:
@sw1 It would be great to have you contribute the discussion of these papers.  They appear to fit better with the Categorize section instead of this Study section.  Can you please review the [Categorize section stubs](https://github.com/greenelab/deep-review/blob/master/sections/03_categorize.md) and create a separate pull request with your edits?

agitter commented:
@cgreene Do you have any comments on the global organization of this section before we go further outlining the details?  Is the data type-focused division of subsections likely to lead to too much of an "enumerative" style?

gailrosen commented:
I don't think I can make a Dec. 1 deadline... but I will try to make a Dec.
20th deadline.

On Sun, Nov 13, 2016 at 8:28 AM, Anthony Gitter <notifications@github.com>
wrote:

> @cgreene <https://github.com/cgreene> Do you have any comments on the
> global organization of this section before we go further outlining the
> details? Is the data type-focused division of subsections likely to lead to
> too much of an "enumerative" style?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/greenelab/deep-review/pull/136#issuecomment-260186223>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE-YubzZylNR4JhQr_i2w1Mk0UX1dDNlks5q9xBygaJpZM4Ksg1K>
> .
>



-- 
Gail L. Rosen, Associate Professor
Electrical and Computer Engineering
Drexel University
Webpage/Contact info:  http://www.ece.drexel.edu/gailr

cgreene commented:
I am in the same boat. Have a grant going out on the 12th that has greatly
impacted my available time

On Tue, Nov 29, 2016, 5:33 PM gailrosen <notifications@github.com> wrote:

> I don't think I can make a Dec. 1 deadline... but I will try to make a Dec.
> 20th deadline.
>
> On Sun, Nov 13, 2016 at 8:28 AM, Anthony Gitter <notifications@github.com>
> wrote:
>
> > @cgreene <https://github.com/cgreene> Do you have any comments on the
> > global organization of this section before we go further outlining the
> > details? Is the data type-focused division of subsections likely to lead
> to
> > too much of an "enumerative" style?
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/greenelab/deep-review/pull/136#issuecomment-260186223>,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AE-YubzZylNR4JhQr_i2w1Mk0UX1dDNlks5q9xBygaJpZM4Ksg1K
> >
> > .
> >
>
>
>
> --
> Gail L. Rosen, Associate Professor
> Electrical and Computer Engineering
> Drexel University
> Webpage/Contact info: http://www.ece.drexel.edu/gailr
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/greenelab/deep-review/pull/136#issuecomment-263722032>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAhHs4WFAAhQTkN-o7t049TYWF2JKiF_ks5rDKg0gaJpZM4Ksg1K>
> .
>

cgreene commented:
Ok. My grant is in, and I am back to the world of the able to participate! @gailrosen still on board?

gailrosen commented:
yeah.. kind of busy and leave on Wednesday for Florida, but I can try to
work on this over the break.

On Mon, Dec 19, 2016 at 9:42 AM, Casey Greene <notifications@github.com>
wrote:

> Ok. My grant is in, and I am back to the world of the able to participate!
> @gailrosen <https://github.com/gailrosen> still on board?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/greenelab/deep-review/pull/136#issuecomment-267980176>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE-YuSnwJThbzplvGxf1EH8ziMOH37fjks5rJpfEgaJpZM4Ksg1K>
> .
>



-- 
Gail L. Rosen, Associate Professor
Electrical and Computer Engineering
Drexel University
Webpage/Contact info:  http://www.ece.drexel.edu/gailr

cgreene commented on sections/04_study.md:
```diff
@@ -0,0 +1,92 @@
+*Predicting gene expression levels and unsupervised approaches for learning
+from gene expression.  Those could be divided into separate sub-sections.*
+
+### Splicing
```
I like splicing as separate from gene expression, unless we change things to "transcript expression"

cgreene commented on sections/04_study.md:
```diff
@@ -0,0 +1,92 @@
+can be discussed with the single-cell papers.*
+
+### Metagenomics
+
```
Do you want to update this to confirm that there will be a section? @gailrosen has sufficient content to fill out a section.

XieConnect commented:
@agitter  I can resume from where we left behind last month if help is still needed on this section (your comment at https://github.com/greenelab/deep-review/pull/136#issuecomment-260185929). 
Otherwise, I'll move on to the EHR-related sections. Thanks for your detailed instructions. 

(Sorry that for some reason, I didn't always get targeted notifications addressed to me and they often got buried among other general discussions.)

agitter commented:
@XieConnect Yes, I would still like help with this section.  However, if you prefer to start on EHR content first, that's also great.

cgreene commented on sections/04_study.md:
```diff
@@ -0,0 +1,90 @@
+expertise, in which case we may acknowledge the application area but not
+dive into merits or weaknesses of individual methods.*
+
+### Gene expression
```
If you want to assign these bits out like you did with the metagenomics section, you can assign this and splicing to me. I may see if we can snag a splicing guru from Penn (Yoseph Barash). If not, I did read a few of those papers and could write that bit.

agitter commented:
@cgreene I confirmed the metagenomics and splicing sub-sections per your comments above.  I also removed methylation and added variant calling to the sequencing sub-section.  I think we'll want to discuss #99 and #159.  Going to merge now.

@sw1 before closing this pull request, I wanted to remind you of the three papers you outlined above for the categorize section.

XieConnect commented:
@agitter I can help finish this section first, and then move on to other sections. I'll start reading through related thread discussions here and existing literature (especially prior reviews) and present a draft for you. I'll try to finish early this week. Thanks a lot.
